index.html

```diff
@@ -34,5 +34,154 @@ <h2 class="welcoming">Hey there. I’m Anas
+
+      <div class="card card2">
+         <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+      </div>
+      <div class="card card3">
+         <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+      </div>
+      <div class="card card4">
+         <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+      </div>
+      <div class="card card5">
+         <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+      </div>
+      <div class="card card6">
+         <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+      </div>
+   </section>
+   <section class="about-myself">
+      <div>
+      <h3>About My Self</h3>
+      <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+         don’t hestiate to contact me.</p>
+         <button type="button">Get My Resume</button>
+      </div>   
+      <div class="a-card">
+         <img src="images/icon-languages.png" alt="">
+         <h5>Languages</h5>
+         <ul>
+            <li>JavScript</li>
+            <li>Ruby</li>
+            <li>html</li>
+            <li>css</li>
+         </ul>
+      </div>
+      <div class="a-card">
+         <img src="images/icon-frameworks.png" alt="">
+         <h5>Frameworks</h5>
+         <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+         </ul>
```

Comment:
- [ ] please try to stick to the Figma design  as much as possible, you should use the right color for technologies cards in order to match the Figma design 
![image](https://user-images.githubusercontent.com/61790587/171466042-7d01c206-070f-404b-ba78-4620d4ccf2de.png)

Reply:
Hi, @youmari  this color was forced by webhint if I use any other color a error occurs I've tried multiple times

Reply:
> Hi, @youmari this color was forced by webhint if I use any other color a error occurs I've tried multiple times

 @ChaabaneAnas you should follow the requirements and your design should match the Figma design and if and error appear try to fix that error